ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py

```diff
@@ -211,3 +211,26 @@ def arctan2(
 
 
 arctan2.unsupported_dtypes = {"torch": ("float16",)}
+
+
+@from_zero_dim_arrays_to_float
+def radians(
+    x,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting='same_kind',
+    order='K',
+    dtype=None,
+    subok=True
+):
+    if dtype:
+        x = ivy.astype(ivy.array(x), ivy.as_ivy_dtype(dtype))
+    ret = ivy.divide(ivy.multiply(x, ivy.pi, out=out), 180)
+    if ivy.is_array(where):
+        ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
+    return ret
+
+
+radians.unsupported_dtypes = {"torch": ("float16",)}
```

Comment:
I don't think this is unsupported according to the backend, please remove this line

Reply:
@juliagsy  thanks for the review, let me remove it